### PR TITLE
add pow and sqrt

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Require https://github.com/neo-project/neo/pull/2359 .
Close https://github.com/neo-project/neo-devpack-dotnet/issues/471

### CODE REVIEW

**Review by shargon — DISMISSED** [review]

File: src/Neo.SmartContract.Framework/Helper.cs [hunk 1]
```diff
@@ -197,5 +197,90 @@ public static byte ToByte(this int source)
...
         public extern static UInt160 ToScriptHash(this string address);
+
+        /// <summary>
+        /// Return the value of x to the power of exponent
+        /// </summary>
+        /// <param name="x">base</param>
+        /// <param name="exponent"></param>
+        /// <returns></returns>
+        [OpCode(OpCode.POW)]
+        public extern static BigInteger Pow(this int x, int exponent);
```
Comment (erikzhang) [inline src/Neo.SmartContract.Framework/Helper.cs:208]: Maybe we can remove these overloads and use `BigInteger.Pow()` uniformly.

File: src/Neo.SmartContract.Framework/Helper.cs [hunk 2]
```diff
@@ -197,5 +197,90 @@ public static byte ToByte(this int source)
...
+        [OpCode(OpCode.SQRT)]
+        public extern static BigInteger Sqrt(this ulong x);
+
+        /// <summary>
+        /// Returns the square root of number x
+        /// </summary>
+        /// <param name="x"></param>
+        /// <returns></returns>
+        [OpCode(OpCode.SQRT)]
+        public extern static BigInteger Sqrt(this BigInteger x);
```
Comment (erikzhang) [inline src/Neo.SmartContract.Framework/Helper.cs:284]: Maybe we can keep this method only.

Comment (shargon) [reply src/Neo.SmartContract.Framework/Helper.cs:216]: Done

File: src/Neo.SmartContract.Framework/Helper.cs (hunk 1 above)
Comment (shargon) [reply src/Neo.SmartContract.Framework/Helper.cs]: Done

**Review by shargon — DISMISSED** [review]

File: src/Neo.SmartContract.Framework/Helper.cs [hunk 3]
```diff
@@ -197,5 +197,22 @@ public static byte ToByte(this int source)
...
         public extern static UInt160 ToScriptHash(this string address);
+
+        /// <summary>
+        /// Return the value of x to the power of exponent
+        /// </summary>
+        /// <param name="x">base</param>
+        /// <param name="exponent"></param>
+        /// <returns></returns>
+        [OpCode(OpCode.POW)]
+        public extern static BigInteger Pow(this BigInteger x, int exponent);
```
Comment (erikzhang) [inline src/Neo.SmartContract.Framework/Helper.cs:208]: This can be removed too. We can use `BigInteger.Pow(x, exponent)`.

Comment (shargon) [reply src/Neo.SmartContract.Framework/Helper.cs:208]: Done

**Review by shargon — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**